.github/workflows/build_and_push_docker_images.yaml

```diff
@@ -183,9 +183,19 @@ jobs:
           img_suffix="elixir-${{ matrix.arch }}"
           img_labels="org.opencontainers.image.elixir.version=${{ matrix.elixir }}\n${img_labels}"
         fi
+
+        if [ ${{ matrix.profile }} = "emqx" ]; then
+          img_labels="org.opencontainers.image.edition=Opensource\n${img_labels}"
```

Comment:
@zmstone I'm not sure we should use `Opensource` or `opensource`,  do you have any ideas ?

Reply:
Opensource

Reply:
OK